### PR TITLE
feat: add menu management page

### PR DESCRIPTION
**Pull request by wu-clan**
- [x] Basic display page
- [x] Inquiry function
- [x] Expand / Collapse
- [x] New page
- [x] Edit page
- [x] Delete page
- [x] New function
- [x] Edit function
- [x] Delete function

### CODE REVIEW

File: src/router/routes/modules/menu.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { AppRouteRecordRaw } from '@/router/routes/types';
+import { DEFAULT_LAYOUT } from '@/router/routes/base';
+
+const SYSTEM: AppRouteRecordRaw = {
+  path: '/admin',
+  name: 'admin',
```
Comment (downdawn) [inline src/router/routes/modules/menu.ts:6]: admin? not menu?

**Review by downdawn — APPROVED** [review]

**Comment by downdawn** [comment]
LGTM.

File: src/router/routes/modules/menu.ts (hunk 1 above)
Comment (wu-clan) [reply src/router/routes/modules/menu.ts:6]: This should be the root directory, which is the system administration, and the lower level contains the menu administration

Comment (wu-clan) [reply src/router/routes/modules/menu.ts:6]: This should be the root directory, which is the system management, and the lower level contains the menu management

Comment (wu-clan) [reply src/router/routes/modules/menu.ts:6]: This corresponds to the directory in views, I called it system at first, then changed to admin (similar to go-admin)

Comment (downdawn) [reply src/router/routes/modules/menu.ts:6]: ok. generally system

**Pull request merged**